packages/wrangler/src/deployment-bundle/worker.ts

```diff
@@ -111,6 +111,7 @@ export interface CfBrowserBinding {
 
 export interface CfAIBinding {
 	binding: string;
+	staging?: boolean;
```

Comment:
Do we want to mark this with a JSDoc `@internal` attribute or some other way to clarify to external users that they shouldn't use this?

Reply:
We already have `first_party_worker`, `logfwdr` and `unsafe` bindings so no need to block this PR now. We can follow up later if this is applicable.